snap/snapcraft.yaml

```diff
@@ -8,11 +8,11 @@ description: |
 
 grade: stable
 confinement: strict
-base: core18
+base: core20
```

Comment:
Hm, probably better will be use latest core component. (core24)
This will be works on all older distribution include Ubuntu 18.04, but we can get access to new feature of snap. 

You can live core20, but it will be deprecated soon

Reply:
Okay, it's real more important for qt6, qt5 depends on older system libraries.